modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,211 @@
+
+--
+
+======
+
+When the mount operation is complete, the target cluster handles produce and consume workloads for the topics.
+
+== Cancel a mount or unmount operation
+
+You can cancel a topic mount or unmount by running the command:
+
+[tabs]
+======
+rpk::
++
+--
+```
+rpk cluster storage cancel-mount <migration-id>
+```
+--
+
+Admin API::
++
+--
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=cancel
+```
+--
+======
+
+You cannot cancel mount and unmount operations in the following <<monitor-progress,states>>:
+
+- `planned` (but you may still xref:api:ROOT:admin-api.adoc#delete-/v1/migrations/-id-[delete] a planned mount or unmount)
+- `cut_over`
+- `finished`
+- `canceling`
+- `cancelled`
+
+== Monitor progress
+
+[tabs]
+======
+rpk::
++
+--
+To list active mount and unmount operations, run the command:
+
+```
+rpk cluster storage list-mount
+```
+--
+
+Admin API::
++
+--
+Issue a GET request to the `/migrations` endpoint to view the status of topic mount and unmount operations:
+
+```
+curl http://localhost:9644/v1/migrations 
+```
+--
+======
+
+The response returns the IDs and state of existing mount and unmount operations ("migrations"):
+
+|===
+| State | Unmount operation (outbound) | Mount operation (inbound)
+
+| `planned`
+2+| Redpanda validates the operation definition.
+
+
+| `preparing`
+| Redpanda flushes topic data, including topic manifests, to the destination bucket or container in object storage.
+| Redpanda recreates the topics in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifests found in object storage.
+
+| `prepared` 
+| The operation is ready to execute. In this state, the cluster still accepts client reads and writes for the topics.
+| Topics exist in the cluster but clients do not yet have access to consume or produce.
+
+| `executing` 
+| The cluster rejects client reads and writes for the topics. Redpanda uploads any remaining topic data that has not yet been copied to object storage. Uncommitted transactions involving the topic are aborted.
+| The target cluster checks that the topic to be mounted has not already been mounted in any cluster.
+
+| `executed` 
+| All unmounted topic data from the cluster is available in object storage.
+| The target cluster has verified that the topic has not already been mounted.
+
+| `cut_over`
+| Redpanda deletes topic metadata from the cluster, and marks the data in object storage as available for mount operations.
+| The topic data in object storage is no longer available to mount to any clusters.
+
+| `finished`
+| The operation is complete and then deleted. 
+| The operation is complete and then deleted. The target cluster starts to handle produce and consume workloads.
+
+| `canceling`
+2+| Redpanda is in the process of canceling the operation.
+
+
+| `cancelled`
+2+| The operation is canceled.
```

Comment:
you have two different spellings of canceled here.

Reply:
as "cancelled" is already a state in the code, suggest that you do "cancelled" in the description too.